background.js

```diff
@@ -1,16 +1,31 @@
 const oldReddit = "https://old.reddit.com";
+const excludedSubdomains = [
+  "blog.reddit.com",
+  "mod.reddit.com",
+  "oauth.reddit.com",
+  "out.reddit.com"];
+const excludedPaths = [
+  "/chat",
```

Comment:
https://old.reddit.com/chat loads absolutely fine

Reply:
My bad, it was a connection issue